### PR TITLE
Package version bump and rename

### PR DESCRIPTION
**Pull request by carlodri**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
@conda-forge/core I need to rename this feedstock from `sk-video` to  `scikit-video` (`sk-video` is an old name which is now discontinued): can someone help me on the procedure?

xref: https://github.com/scikit-video/scikit-video/issues/110

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% set name = "sk-video" %}
-{% set version = "1.1.10" %}
-{% set sha256 = "fb7125cfa2b31253dcf181d0bf285edd6bb25ffebf61dbd8ba11fadb783609a1" %}
+{% set name = "scikit-video" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:1]: Just this is enough. We don't need to rename the feedstock to get the packaged renamed.

However, if you want to solve that inconsistency, you can re-submit to staged-recipes the recipe and we can archive this feedstock.

Comment (carlodri) [reply recipe/meta.yaml:1]: so there's no way to rename the feedstock? can't the admins simply change the feedstock name? Or are there other subtle issues in doing this?

Comment (ocefpaf) [reply recipe/meta.yaml:1]: > Or are there other subtle issues in doing this?

We can change the name but there are some automated services that look for it and may break. I believe the new bot is imune to that but not the webservices.

TL;DR re-submitting is the "cleanest" way to do it. (If you don't care about the feedstock name this PR is OK too and a new package with the new name will be uploaded.)

Comment (carlodri) [reply recipe/meta.yaml:1]: ok, thanks @ocefpaf, my OCD prefers the "clean" way 😀.... I will resubmit then to staged-recipies and ask for deletion of this feedstock in the PR.

Comment (ocefpaf) [reply recipe/meta.yaml:1]: Cool. Feel free to ping me over at staged-recipes.

PS: we don't delete the feedstock, we only archive them so they stop accepting PRs and issues.

**Comment by carlodri** [comment]
closing since a new feedstock will be created replacing this one (https://github.com/conda-forge/staged-recipes/pull/6700)
